package.json

```diff
@@ -10,6 +10,7 @@
   },
   "preferGlobal": true,
   "files": [
+    "bin/.fast-boot.json",
```

Comment:
Does this not get covered by the `bin/` below?

Reply:
It may indeed be redundant!